dbus-gridmeter_sml.py

```diff
@@ -86,13 +91,15 @@ def __init__(self, port, servicename, deviceinstance, paths, productname='Smartm
 
 
     def _getSmartMeterSerial(self):
-        # TODO
-        return 12345
+        meter_data = self._getSmlSmartmeterData(True)
+        if meter_data is None or not meter_data:
+            return None
+        return "%s %s" % (meter_data['mfg'],meter_data['serial'])
```

Comment:
on my meter, this returns a hex string, not the value written on the front. Is this the same with yours? maybe one should convert it directly then

Reply:
to be clear, it is the correct value, just in hex and thus meaningless to the user